### PR TITLE
Lesson1, test.py and integration testing 

### PR DESCRIPTION
**Pull request by pkoleyni**

### CODE REVIEW

File: students/pooria_k/lesson01/inventory_management/main.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    while user_prompt not in valid_prompts:
+        options_str = ("{}" + (", {}") * (len(options)-1)).format(*options)
+        print("Please choose from the following options ({}):".format(options_str))
+        print("1. Add a new item to the inventory")
+        print("2. Get item information")
+        print("q. Quit")
+        user_prompt = input(">")
+    return valid_prompts.get(user_prompt)
+
+def get_price(item_code):
```
Comment (ldconejo) [inline students/pooria_k/lesson01/inventory_management/main.py:26]: Hi Pooria,

It seems this one was modified and the functionality of the function changed. It was originally like this:

```
def get_price(item_code):
    """Displays price of an item"""
    price = market_prices.get_latest_price(item_code)
    return price
```

Could you restore the function and update your unit testing accordingly?

Thanks,

Luis

File: students/pooria_k/lesson01/inventory_management/main.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    """Get price"""
+    print("Get price")
+
+def addnew_item():
+    """Add new Item"""
+    global FULL_INVENTORY
```
Comment (ldconejo) [inline students/pooria_k/lesson01/inventory_management/main.py:32]: Any reason you for adding this line? The use of _global_ will be flagged by Pylint.

**Comment by ldconejo** [comment]
Hi Pooria,

Take a look at the comments in your code and implement the fixes requested. Also, make sure your code is graded by Pylint with 10 out 10 (using Pylint's selective disabling of warnings is ok as long as the warning cannot be corrected by code changes). Let me know if you have any questions.

Regards,

Luis

File: students/pooria_k/lesson01/inventory_management/main.py (hunk 1 above)
Comment (pkoleyni) [reply students/pooria_k/lesson01/inventory_management/main.py:26]: Hi Luis 
I got the file from 
https://github.com/UWPCE-PythonCert-ClassRepos/SP_Python220B_2019/blob/master/lessons/lesson01/assignment/inventory_management/main.py#L23

def getPrice(itemCode):
    print("Get price")

Is that the right repo to get the files for assignment ?

File: students/pooria_k/lesson01/inventory_management/main.py (hunk 2 above)
Comment (pkoleyni) [reply students/pooria_k/lesson01/inventory_management/main.py:32]: I might be wrong but i got the files from following repo 
https://github.com/UWPCE-PythonCert-ClassRepos/SP_Python220B_2019/blob/master/lessons/lesson01/assignment/inventory_management/main.py#L27

I didnt add global variable.
Is that the right repo to get the files for assignment ?

**Pull request closed without merging**